.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "zxing-cpp"]
+	path = zxing-cpp
+	url = https://github.com/nu-book/zxing-cpp
```

Comment:
Use
https://git.sailfishos.org/mirror/zxing-cpp

Reply:
I guess it's only used for packages hosted on git.sailfishos.org

Reply:
Haven't had too many mirroring projects yet here so confirmed: should use the sailfish mirror also on github repositories.